ignite/metadata.csv

```diff
@@ -183,14 +183,14 @@ ignite.cache.entry_processor.hits,rate,10,,,"The total number of invocations on
 ignite.cache.entry_processor.hit_percentage,gauge,10,percent,,"The percentage of invocations on keys, which exist in cache.",1,ignite,
 ignite.cache.entry_processor.misses,rate,10,,,"The total number of invocations on keys, which don't exist in cache.",-1,ignite,
 ignite.cache.entry_processor.miss_percentage,gauge,10,percent,,"The percentage of invocations on keys, which don't exist in cache.",-1,ignite,
-ignite.cache.entry_processor.average_invocation_time,gauge,10,second,,The mean time to execute cache invokes.,-1,ignite,
-ignite.cache.entry_processor.minimum_invocation_time,gauge,10,second,,"So far, the minimum time to execute cache invokes.",-1,ignite,
-ignite.cache.entry_processor.maximum_invocation_time,gauge,10,second,,"So far, the maximum time to execute cache invokes.",-1,ignite,
+ignite.cache.entry_processor.average_invocation_time,gauge,10,microsecond,,The mean time to execute cache invokes.,-1,ignite,
+ignite.cache.entry_processor.minimum_invocation_time,gauge,10,microsecond,,"So far, the minimum time to execute cache invokes.",-1,ignite,
+ignite.cache.entry_processor.maximum_invocation_time,gauge,10,microsecond,,"So far, the maximum time to execute cache invokes.",-1,ignite,
 ignite.cache.total_partitions,gauge,10,,,Total number of partitions on current node.,0,ignite,
 ignite.cache.rebalancing_partitions,gauge,10,,,Number of currently rebalancing partitions on current node.,0,ignite,
 ignite.cache.rebalanced_keys,gauge,10,key,,Number of already rebalanced keys.,0,ignite,
 ignite.cache.estimated_rebalancing_keys,gauge,10,key,,Number estimated to rebalance keys.,0,ignite,
 ignite.cache.keys_to_rebalance,gauge,10,key,,Estimated number of keys to be rebalanced on current node.,0,ignite,
-ignite.cache.rebalancing_keys_rate,gauge,10,byte,second,Estimated rebalancing speed in keys.,0,ignite,
+ignite.cache.rebalancing_keys_rate,gauge,10,entry,second,Estimated rebalancing speed in keys.,0,ignite,
```

Comment:
A better suggestion for this? It's not bytes that's for sure

Reply:
`occurrence` `operation` `sample` `execution` seem like good options as well to describe rebalancing keys.